### DISCUSSION TITLE
Post Your Ideas Here!

### DISCUSSION THREAD

**Discussion by hamdivazim-archive** [hamdivazim/CodeGlob#2] [Ideas]
## Post Ideas!
If you think you have a _huge_  idea for CodeGlob, [start a new discussion](https://github.com/hamdivazim/CodeGlob/discussions/new?category=ideas) so we can add to it, and so you make yourself known :)

If you don't have a major idea or you have a minor bug fix, that's absolutely fine! Just comment down below this discussion and the dev team will work on fixing it :)

Keep _'Globbing'_!

**Comment by MKM12345** [comment]
I was thinking about a nice-looking, animated CSS login form to enter the code blob initially

**Reply by MKM12345** [reply]
Although I'm not sure about the PHP part

**Reply by MKM12345** [reply]
Something like this:https://github.com/rootintootinwar/neon-login-register-form
